### PR TITLE
Handle USDT and BUSD

### PR DESCRIPTION
**Pull request by s0undt3ch**
I'm not saying this code is perfect, but it handles USDT and BUSD

### CODE REVIEW

File: src/futuresboard/blueprint.py [hunk 1]
```diff
@@ -310,19 +331,19 @@ def index_page():
...
     by_symbol = db.query(
         'SELECT SUM(income) AS inc, symbol FROM income WHERE asset <> "BNB" AND incomeType NOT IN'
-        + " ({0})".format(", ".join("?" for _ in remove_incomeTypes))
+        + " ({})".format(", ".join("?" for _ in remove_incomeTypes))
         + " AND time >= ? AND time <= ? GROUP BY symbol ORDER BY inc DESC",
         remove_incomeTypes + [start, end],
     )
 
-    fees = {"USDT": 0, "BNB": 0}
+    fees = {"USDT": 0, "BNB": 0, "BUSD": 0, "BUSD": 0}
```
Comment (ecoppen) [inline src/futuresboard/blueprint.py:346]: Is this supposed to be here twice?

Comment (s0undt3ch) [reply src/futuresboard/blueprint.py:346]: Woops, no.

Comment (s0undt3ch) [reply src/futuresboard/blueprint.py:346]: Doesn't hurt, but is bad python :)

**Comment by ecoppen** [comment]
Will implement in v2

**Pull request closed without merging**